
\RequirePackage{fix-cm}
\RequirePackage[hyphens]{url}
\RequirePackage[final]{graphicx} % need to show figures in draft mode
\documentclass[aps,pre,twocolumn,nofootinbib,superscriptaddress,10pt, final,tightenlines]{revtex4-1}


%\usepackage[numbers]{natbib}

%Todonotes package
\usepackage[obeyFinal]{todonotes}

%Configure todonotes to set it up the way we want
% Comments
\newcounter{comment}
\newcommand{\comment}[2][]{%
% initials of the author (optional) + note in the margin
\refstepcounter{comment}%
{%
\setstretch{0.7}% spacing
\todo[inline, color={cyan!45},size=\small]{%
\textbf{\footnotesize [\uppercase{#1}\thecomment]:}~#2}%
}}


\newcommand{\insertref}[1]{\todo[color=green!40]{#1}}
\newcommand{\adddetail}[1]{\todo[color=blue!20]{#1}}
\newcommand{\major}[1]{\todo[color=red]{#1}}

\usepackage[colorlinks=true,citecolor=blue,linkcolor=blue]{hyperref}

\usepackage{url}
\usepackage{graphicx}

\setcounter{secnumdepth}{4}


%Reduce spacing between list items
\usepackage{enumitem}
\setlist{nolistsep}

\usepackage{siunitx}
\sisetup{separate-uncertainty=true}
\AtBeginDocument{\usepackage{booktabs}}

\begin{document}


\title{Predicting binding free energies: Frontiers and benchmarks}


%Authors, affiliations, address.
\author{David L. Mobley}
\email{dmobley@mobleylab.org}
\affiliation{Departments of Pharmaceutical Sciences and Chemistry, University of California, Irvine, CA, USA, 92697}
\author{Michael K. Gilson}
\email{mgilson@ucsd.edu}
\affiliation{Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego, CA, USA, 92092}


\begin{abstract}
Binding free energy calculations based on molecular simulations provide predicted affinities for biomolecular complexes. 
These calculations begin with a detailed description of a system, including its chemical composition and the interactions between its components. 
Simulations of the system are then used to compute thermodynamic information, such as binding affinities. Because of their promise for guiding molecular design, these calculations have recently begun to see widespread applications in early stage drug discovery. However, many challenges remain to make them a robust and reliable tool. Here, we briefly explain how the calculations work, highlight key challenges, and argue for the development of accepted benchmark test systems that will help the research community generate and evaluate progress.
\end{abstract}

\maketitle



\section{INTRODUCTION}

Molecular simulations provide a powerful technique for predicting and understanding the structure, function, dynamics, and interactions of biomolecules. 
Often, these techniques are valued because they provide a movie of what might be going on at the atomic level. 
However, simulations also can be used to make quantitative predictions of thermodynamic and kinetic properties, with applications in fields including drug discovery, chemical engineering, and nanoengineering. 
A thermodynamic property of particular interest is the binding affinity between biomolecules and ligands such as inhibitors, modulators, or activators. 
With accurate and rapid affinity predictions, we could use simulations in varied health-related applications, such as the prediction of biomolecular interaction networks in support of systems biology, or rapid design of new medications with reduced side-effects and drug resistance.

\subsection{Imagining a tool for drug discovery}
A major aim in the development of molecular simulations is to create quantitative, accurate tools which will guide early stage drug discovery. 
Consider a medicinal chemist in the not-too-distant future who has just finished synthesizing several new derivatives of an existing inhibitor as potential drug leads targeting a particular biomolecule, and has obtained binding affinity or potency data against the desired biomolecular target. 
Before leaving work, he or she generates ideas for perhaps 100 new compounds which could be synthesized next, then sets a computer to work overnight prioritizing them. 
By morning, the compounds have all been prioritized based on reliable predictions of their affinity for the desired target, selectivity against alternative targets which should be avoided, solubility, and membrane permeability.  
The chemist then looks through the predicted properties for the top few compounds and selects the next ones for synthesis. 
If synthesizing and testing each compound takes several days, this workflow compresses roughly a year's work into a few days.


While this workflow is not yet a reality, huge strides have been made in this direction, with calculated binding affinity predictions now showing real promise~\cite{mobley_perspective_2012, christ_accuracy_2014, deng_distinguishing_2015, sherborne_preprint_2016, schrodinger_accurate_2015, christ_binding_2016, cui_affinity_2016, verras_free_2016}, solubility predictions beginning to come online~\cite{Schnieders:2012:J.Chem.TheoryComput., park_absolute_2014, liu_using_2016}, and predicted drug resistance/selectivity also apparently tractable~\cite{leonis_contribution_2013, leonis_contribution_2013}, with some headway apparent on membrane permeability~\cite{lee_permeability_2016, comer_permeability_2014}. 
A considerable amount of science and engineering still remains to make this vision a reality, but, given recent progress, the question now seems more one of \emph{when} rather than \emph{whether}. 

\subsection{Increasing accuracy will yield increasing payoffs}

Recent progress in computational power, especially the widespread availability of graphics processing units (GPUs) and advances in automation~\cite{liu_lead_2013} and sampling protocols, have helped simulation-based techniques reach the point where they now appear to have sufficient accuracy to be genuinely useful in guiding pharmaceutical drug discovery at least for a certain subset of problems~\cite{mikulskis_large-scale_2014, homeyer_binding_2014, sherborne_preprint_2016,  schrodinger_accurate_2015, christ_binding_2016, cui_affinity_2016, verras_free_2016}.
Specifically, in some situations, free energy calculations appear to be capable of achieving RMS errors in the 1-2 kcal/mol range with current force fields, even in prospective applications.
As a consequence, pharmaceutical companies are beginning to use these methods in discovery projects. 
The most immediate application of these techniques is to guide synthesis for lead optimization, but applications to scaffold hopping and in other areas also appear possible.

At the same time, it is clear that not all situations are so favorable, so it is worth asking what level of accuracy is actually needed. 
It is often suggested that we need binding free energy predictions accurate to within $\sim 1$ kcal/mol, but we are not aware of a clear basis for this figure beyond the fact it is a pleasingly round number that is close to the thermal kinetic energy, $RT$. 
Instead of setting a single threshold requirement for accuracy, it is more informative to consider how accurate calculations must be to reduce the number of compounds synthesized and tested by some factor, relative to the number required without computational prioritization. 
If one targets a three-fold reduction, the answer appears to be that calculations with a 2 kcal/mol RMS error will suffice~\cite{shirts_free-energy_2010, mobley_perspective_2012}. 
Thus, one can gain substantial benefit from simulations that are good yet still quite imperfect.

% Following paragraph could be cut for space reasons if needed.
More broadly, this analysis does not address the net value of computational affinity predictions in drug discovery.
Costs include those of the software, computer time, and personnel required to incorporate calculations into the workflow; while benefits include the savings, revenue gains, and externalities attributable to reducing the number of low-affinity compounds synthesized and arriving earlier at a potent drug candidate.
In addition, with sufficiently reliable predictions, chemists may choose to tackle difficult synthesis efforts they otherwise might have avoided, resulting in more novel and valuable chemical matter.

\subsection{Overview of free energy calculations}
\label{sec:FEMethods}
The present review focuses on a class of methods in which free energy differences are computed with simulations that sample Boltzmann distributions of molecular configurations. 
These samples are usually generated by molecular dynamics (MD) simulations~\cite{karplus_molecular_2002}, with the system effectively coupled to a heat bath at constant temperature, but Monte Carlo methods may also be used~\cite{michel_hit_2008, michel_prediction_2010, cole_molecular_2015}.  
In either case, the energy of a given configuration is provided by a potential function, or force field, which estimates the potential energy of a system of solute and solvent molecules as a function of the coordinates of all of its atoms. 
Such simulations may be used in several different ways to compute binding free energies or relative binding free energies, as detailed elsewhere~\cite{michel_prediction_2010, christ_basic_2010,  chodera_alchemical_2011, shirts_introduction_2013} and summarized below. In all cases, however, the calculations yield the free energy difference between two states of a molecular system, and they do so by computing the reversible work for changing the initial state to the final one. Two broad approaches deserve mention.

The first general approach directly computes the standard free energy of binding of two molecules by computing the reversible work of transferring the ligand from the binding site into solution. 
(This is sometimes called an absolute binding free energy calculation.) 
The pathway of this change may be one that is physically realizable, or one that is only realizable {\em in silico}, in which case it is sometimes called an ``alchemical'' pathway. 
Physical pathway methods provide the standard binding free energy by computing the reversible work of, in effect, pulling the ligand out of the binding site. 
Although, by definition, the pathway used must be a physical one that could occur in nature, it need not be probable, and improbable pathways, governed by an order parameter specifying how far the ligand is from the binding site, are often used~\cite{woo_calculation_2005, ytreberg_absolute_2009, velez-vega_overcoming_2013, henriksen_computational_2015, hsiao_prediction_2014, bhakat_resolving_2016}.  
In addition, artificial restraints may be useful to avoid sampling problems in the face of often complex barriers along the pathway ~\cite{woo_calculation_2005, velez-vega_overcoming_2013, henriksen_computational_2015, hsiao_prediction_2014, bhakat_resolving_2016}. 
By contrast, alchemical pathway methods artificially decouple the ligand from the binding site and then recouple it to solution from the protein~\cite{jorgensen_efficient_1988, Hermans:1986:Isr.J.Chem., gilson_statistical-thermodynamic_1997, boresch_absolute_2003, mobley_use_2006}.
Although alchemical decoupling methods may avoid clashes of the ligand with the protein that might be problematic in pathway methods for a tight binding site, they still can pose some of the same sampling challenges.
For example, sampling of the unbound receptor must be adequate after the ligand is removed, and water molecules must have time to equilibrate in the vacated binding site.
Given that free energy is a state function, it is not surprising that alchemical and physical pathway approaches yield apparently comparable results when applied to the same systems~\cite{lee_calculation_2006, gumbart_standard_2013, de_ruiter_proteinligand_2013, yin_overview_2016}.  

The second general approach computes the difference between the binding free energies of two different ligands for the same receptor, by computing the work of artificially converting one ligand into another, first in the bound state and then free in solution \cite{Tembe:1984:ComputChem, michel_prediction_2010, christ_basic_2010,  chodera_alchemical_2011}. 
Because these conversions are not physically realizable, such calculations are, again, called alchemical. These calculations can be quite efficient if the two ligands are very similar to each other, but they become more complicated and pose greater sampling problems if the two ligands are very different chemically or if there is a high barrier to interconversion between their most stable bound conformations ~\cite{liu_lead_2013}. 
In addition, there may be concerns about slow conformational relaxation of the protein in response to the change in ligand. 
Nonetheless, alchemical relative free energy calculations currently are the best automated and most widely used free energy methods~\cite{mobley_perspective_2012, liu_lead_2013, schrodinger_accurate_2015}.

Importantly, the accuracy and precision of all of these methods are controlled by the same considerations.
First, many conformations typically need to be generated, or sampled, in order to obtain an adequate representation of the Boltzmann distribution. 
In the limit of infinite sampling, a correctly implemented method would yield the single value of the free energy difference dictated by the specification of the molecular system and the chosen force field. 
In reality, however, only finite sampling is possible, so the reported free energy will differ from the nominal value associated with infinite sampling. 
In addition, because sampling methods are typically stochastic and the dynamics of molecular systems are highly sensitive to initial conditions~\cite{allen_computer_1989}, repeated calculations, using different random number seeds or initial states, will yield different results. 
The problem of finite sampling is most acute for systems where low-energy (hence highly occupied) conformational states are separated by high effective barriers, whether energetic or entropic. 
Second, even if adequate sampling is achievable, free energy differences may disagree substantially with experiment if the force field is not sufficiently accurate. 
Third, errors may also arise if the representation of the system in the simulation does not adequately represent the actual system, e.g. if protonation states are assigned incorrectly and held fixed.  

\subsection{Challenges and the domain of applicability}
Thus, in order for a free energy calculation to be reliable, it must use an appropriate representation of the physical system and an accurate force field, and it must adequately sample the relevant molecular configurations. 
In the case of the more widely used alchemical relative free energy approach, this means that the best results are expected when:\vspace{2mm}
\begin{itemize}
\item a high quality receptor structure is available, without missing loops or other major uncertainties
\item the protonation state of the ligand and binding-site residues (as well as any other relevant residues) can reliably be inferred
\item the ligand binding mode is defined by crystallographic studies and is not expected to change much on modification
\item the receptor does not undergo substantial or slow conformational changes
\item key interactions are expected to be well-described by underlying force fields 
\end{itemize}
\vspace{2mm}

Beyond this domain of applicability---whose dimensions are, in fact, still somewhat vague~\cite{Abel:2016:vertex} --- substantial challenges may be encountered. 
For example, binding free energy calculations for a cytochrome C peroxidase mutant suggest limitations of fixed-charge force fields. In this case, the strength of electrostatic interactions in a buried, relatively nonpolar binding site appears to be overestimated by a conventional fixed-charge force field, likely due to underestimation of polarization effects~\cite{rocklin_blind_2013}. 
Sampling problems are also common, with slow sidechain rearrangements and ligand binding mode rearrangements in model binding sites in T4 lysozyme posing timescale problems unless enhanced or biased sampling methods are carefully applied~\cite{mobley_confine_2007, boyce_predicting_2009, mobley_predicting_2007, jiang_free_2010, gallicchio_binding_2010, wang_achieving_2012}; and larger-scale protein motions induced by some ligands also posing challenges~\cite{boyce_predicting_2009, lim_sensitivity_2016}. 

Although such problems need not prevent free energy calculations from being used, they can require specific adjustment of procedures and parameters based on experience and knowledge of the system at hand.  
Thus, a key challenge for the field is how to use insights from well-studied cases to enable automation and reduce the detailed knowledge of each system required to carry out high quality simulations. 

Troubleshooting is also a major challenge. 
In most cases where calculations diverge substantially from experiment, the reason for the discrepancy is not apparent. 
Is the force field inaccurate? 
Would the results improve with more sampling?  Were protonation states misassigned---or do they perhaps even change on binding? 
There might even be a software bug \cite{eklund_cluster_2016} or a human error in the use of the software. 
As a consequence, it is not clear what steps are most urgently needed to advance the field as a whole.


\section{THE NEED FOR WELL-CHOSEN BENCHMARK SYSTEMS}

Although tests of individual free energy methods are not uncommon today~\cite{mikulskis_large-scale_2014, schrodinger_accurate_2015, christ_binding_2016, cui_affinity_2016, verras_free_2016}, the use of nonoverlapping molecular systems and computational protocols makes it difficult to compare methods on a rigorous basis.
In addition, few studies are designed to identify key sources of error and thereby focus future research and development.
A few molecular systems have now emerged as \emph{de facto} standards for general study (Section~\ref{benchmarks}). 
These selections result in part from two series of blinded prediction challenges (SAMPL~\cite{muddana_sampl4_2014}, and CSAR~\cite{dunbar_csar_2011} followed by D3R~\cite{Gathiaka:2016:JComputAidedMolDes}), which have helped focus the computational chemistry community on a succession of test cases and highlighted the need for methodological improvements. 
However, broader adoption of a larger and more persistent set of test cases is needed. 
By coalescing around a compact set of benchmarks, well chosen to challenge and probe free energy calculations, practitioners and developers will be able to better assess and drive progress in binding free energy calculations. 


\subsection{Benchmark types and applications}
\label{subsec:benchmarktypes}
We envision two classes of benchmark cases: ``hard'' benchmarks, which are simple enough that well-converged results can readily be computed; and ``soft'' benchmarks, for which convincingly converged results cannot readily be generated, but which are still simple enough that concerted study by the community can delineate key issues that might not arise in the simpler ``hard'' cases. 
The following subsections provide examples of how hard and soft benchmark systems may be used to address important issues in free energy simulations.

\subsubsection{Hard benchmarks}

\paragraph{Systems to test software implementations and usage}
It is crucial yet nontrivial to validate that a simulation package correctly implements and applies the desired methods~\cite{shirts_lessons_2016}, and benchmark cases can help with this.  
First, all software packages could be tested for their ability to generate correct potential energies for a single configuration of the specified molecular system and force field. 
These results should be correct to within rounding error and the precision of the physical constants used in the calculations~\cite{shirts_lessons_2016}. 
Similarly, different methods and software packages should give consistent binding free energies when identical force fields are applied with identical simulation setups and compositions. 
The benchmark systems for such testing can be simple and easy to converge, and high precision free energies (e.g., uncertainty $\approx 0.1$ kcal/mol) should serve as a reference. 
Test calculations should typically agree with reference results to within 95\% confidence intervals, from established methods~\cite{shirts_statistically_2008,flyvbjerg_error_1989},
For this purpose, the correctly computed values need not agree with experiment; indeed, experimental results are unnecessary.

\paragraph{Systems to check sampling completeness and efficiency}
\label{sec:hard-sampling}
As discussed above, free energy calculations require thorough sampling of molecular configurations from the Boltzmann distribution dictated by the force field that is employed. 
This sampling is typically done by running molecular dynamics simulations, and for systems as large and complex as proteins, it is difficult to carry out long enough simulations. 
Calculations with inadequate sampling yield results that are imprecise, in the sense that multiple independent calculations with slightly different initial conditions will yield significantly different results, and these ill-converged results will in general be poor estimates of the ideal result obtained in the limit of infinite sampling. 
Advanced simulation methods have been developed to speed convergence~\cite{tai_conformational_2004, shirts_introduction_2013},  but it is not always clear how various methods compare to one another. 
To effectively compare such enhanced sampling methods, we need benchmark molecular systems, parameterized with a force field that many software packages can use, that embody various sampling challenges, such as high dimensionality and energetic and entropic barriers between highly occupied states, but which are just tractable enough that reliable results are available via suitable reference calculations.
Again, experimental data are not required, and the point of comparison may be, at least in part, sampling measures.

\paragraph{Systems to assess force field accuracy}
\label{pgph:accuracy}
Some molecular systems are small and simple enough that current technology allows thorough conformational sampling, and hence well converged calculations of experimental observables. 
This has long been feasible for liquids~\cite{jorgensen_quantum_1981}; for example, it is easy to precisely compute the heat of vaporization of liquid acetone with one of the standard force fields.
More recently, advances in hardware and software have made it possible to compute binding thermodynamics to high precision for
simple molecular recognition systems~\cite{henriksen_computational_2015}, as further discussed below. 
In such cases, absent complications like uncertain protonation states, the level of agreement with experiment reports directly on the accuracy of the force field.
Thus, simple molecular recognition systems with reliable experimental binding data represent another valuable class of benchmarks.
Here, of course, experimental data are needed. Ideally, the physical materials will be fairly easy to obtain so that measurements can be replicated or new experimental conditions (such as temperature and solvent composition) explored.

\subsubsection{Soft benchmarks}
\paragraph{Systems to challenge conformational sampling techniques}
Enhanced sampling techniques (Section ~\ref{sec:hard-sampling}), designed to speed convergence of free energy simulations,
may not be adequately tested by any hard benchmark, because such systems are necessarily rather simple. 
Thus, despite the fact that reliable reference results are not available for soft benchmarks, they are still important for method comparisons. 
For example, it may become clear that some methods are better at sampling in systems with high energy barriers, and others in high-dimensional systems with rugged energy surfaces. 
Developers should test methods on a standard set of benchmark systems for informative comparisons.

\paragraph{Direct tests of protein-ligand binding calculations}
Although it is still very difficult to convincingly verify convergence of many protein-ligand binding calculations, it is still important to compare the performance of various methods in real-world challenges. 
Appropriate soft benchmarks are likely to be cases which are still relatively tractable, involving small proteins and simple binding sites.
We need a series of benchmark protein-ligand systems that introduce various challenges in a well-understood manner. 
Systems should introduce none, one, two, or $N$ of the following challenges in various combinations:
\begin{enumerate}
\item Sampling challenges
  \begin{enumerate}
  \item Sidechains in the binding site rearrange on binding different ligands
  \item Modest receptor conformational changes, such as loop motion
  \item Large scale conformational changes, such as domain motions and allostery
  \item Ligand binding modes change unpredictably with small chemical modifications
  \item High occupancy water sites rearrange depending on bound ligand
  \end{enumerate}
\item System challenges
  \begin{enumerate}
  \item Protonation state of ligand and/or protein changes on binding
  \item Multiple protonation states of the ligand and/or receptor are relevant
  \item Results are sensitive to buffer, salts or other environmental factors
\end{enumerate}
\item Force field challenges
  \begin{enumerate}
  \item Strong electric fields suggest that omission of explicit electronic polarizability will limit accuracy
  \item Ligands interact directly with metal ions
  \item Ligands or co-factors challenge existing force fields
\end{enumerate}
\end{enumerate}

\paragraph{Progression of soft benchmarks} We envision these more complex benchmark systems proceeding through stages, initially serving effectively as a playground where major challenges and issues are explored, documented, and become well-known. 
Eventually, some will become sufficiently well characterized and sampled that they become hard benchmarks.

\subsection{Applications and limitations of benchmark systems}
Standard benchmark systems along the lines sketched above will allow potential solutions to be tested in a straightforward, reproducible manner. 
For example, force fields may be assessed by swapping new parameters, or even a new functional form, into an existing workflow to see the impact on accuracy for a hard benchmark test.
Sampling methods may be assessed by using various enhanced sampling methods for either hard or soft sampling benchmarks, here without focusing on accuracy relative to experiment.
And system preparation tools could be varied to see how different approaches to assigning protonation states, modeling missing loops, or setting initial ligand poses, affect agreement with experiment---with the understanding that force field and sampling also play a role. 
Such studies will be greatly facilitated by well-characterized standard benchmarks.

At the same time, there is a possibility that that some methods will inadvertently end up tuned specifically to generate good results for the set of accepted benchmarks. 
In such cases, the results for systems outside the benchmark set might still be disappointing. 
This means the field will need to work together to develop a truly representative set of benchmarks.
This potential problem can also be mitigated by sharing of methods to enable broader testing by non-developers, and by participation in blinded prediction challenges, such as SAMPL and D3R, which confront methods with entirely new challenge cases. 

\section{BENCHMARK SYSTEMS FOR BINDING PREDICTION}
\label{benchmarks}
No molecular systems have been explicitly accepted by the field as benchmarks for free energy calculations, but certain host molecules (see below) and designed binding sites in the enzyme T4 lysozyme have emerged as particularly helpful and widely studied test cases. 
Here, we describe these artificial receptors and propose specific host-guest and T4 lysozyme-ligand combinations as initial benchmark systems for free energy calculations. 
We also point to several additional hosts and small proteins that also have potential to generate useful benchmarks in the future (Section~\ref{sec:updates}). 
The present focus is on cases where experimental data are available and add value, rather than ones chosen specifically to test conformational sampling methods, where experimental data are not required (Section~\ref{subsec:benchmarktypes}). 

\subsection{Host-guest benchmarks}
Chemical hosts are small molecules, often comprising fewer than 100 non-hydrogen atoms, with a cavity or cleft that allows them to bind other compounds, called guests, with significant affinity.  
Hosts bind their guests via the same basic forces that proteins used to bind their ligands, so they can serve as simple test systems for computational models of noncovalent binding. 
Moreover, their small size, and, in many cases, their rigidity, can make it feasible to sample all relevant conformations, making for ``hard" benchmarks as defined above (Section~\ref{subsec:benchmarktypes}).  
Furthermore, experiments can often be run under conditions that make the protonation states of the host and guest unambiguous. Under these conditions, the level of agreement of correctly executed calculations with experiment effectively reports on the validity of the force field (Section~\ref{pgph:accuracy}. 
For a number of host-guest systems, the use of isothermal titration calorimetry (ITC) to characterize binding provides both binding free energies and binding enthalpies. 
Binding enthalpies can often also be computed to good numerical precision~\cite{henriksen_computational_2015}, so they provide an additional check of the validity of simulations. A variety of curated host-guest binding data is available on BindingDB at \url{http://bindingdb.org/bind/HostGuest.jsp}.

Hosts fall into chemical families, such that all members of each family share a major chemical motif, but individuals vary in terms of localized chemical substitutions and, in some families, the number of characteristic monomers they comprise.  
For example, all members of the cyclodextrin family are chiral rings of glucose monomers; family members then differ in the number of monomers and in the presence or absence of various chemical substituents. 
For tests of computational methods ultimately aimed at predicting protein-ligand binding affinities in aqueous solution, water soluble hosts are, arguably, most relevant. 
On the other hand, host-guest systems in organic solvents may usefully test how well force fields work in the nonaqueous environment within a lipid membrane. 
Here, we focus on two host families, the cucurbiturils \cite{freeman_cucurbituril_1981,mock_host-guest_1983}; and the octa-acids (more generally, Gibb deep cavity cavitands)~\cite{gibb_well-defined_2004, hillyer_synthesis_2016}, which have already been the subject of concerted attention from the simulation community, due in part to their use in the SAMPL blinded prediction challenges~\cite{muddana_sampl3_2012, muddana_sampl4_2014, yin_overview_2016}.  

\begin{figure*}
\includegraphics[width=\textwidth]{figures/hosts.pdf}
\caption{\label{hosts} OA, TEMOA, and CB7 hosts. Shown are the hosts which are the focus of our host-guest benchmark sets -- two variants of the octa-acid GDCC, and CB7, a cucurbituril. Guest structures are available in the supplemental material.}
\end{figure*}

\subsubsection{Cucurbiturils}

The cucurbiturils ({\bf Figure~\ref{hosts}}) are achiral rings of glycoluril monomers~\cite{freeman_cucurbituril_1981}. The first characterized family member, cucurbit[6]uril, has six glycoluril units, and subsequent synthetic efforts led to the five-, seven-, eight- and ten-monomer versions, cucurbit[n]uril (n=5,6,7,8,10)~\cite{liu_cucurbituril_2005},
which have been characterized to different extents. 
Of note, the n=6,7,8 variants accommodate guests of progressively larger size, but are consistent in preferring to bind guests with a hydrophobic core sized to fit snugly into the relatively nonpolar binding cavity, along with at least one cationic moiety (though neutral compounds do bind~\cite{wyman_cucurbituril_2008, lee_deciphering_2015}) that forms stabilizing interactions with the oxygens of the carbonyl groups fringing both portals of the host~\cite{liu_cucurbituril_2005}. 
Although derivatives of these parent compounds, have been made \cite{Lee:2003:AccountsofChemicalResearch, vinciguerra_synthesis_2015, assaf_cucurbiturils:_2015, Cong:2016:Org.Biomol.Chem.},
most of the binding data published for this class of hosts pertain to the non-derivatized forms. A fairly extensive set of data is available in BindingDB at \url{http://bindingdb.org/bind/HostGuest.jsp}.


We propose cucurbit[7]uril (CB7) as the basis of one series of host-guest benchmark systems ({\bf Figure~\ref{hosts}}, {\bf Tables~\ref{cb7_benchmark1} and~\ref{cb7_benchmark2}}). 
This host is convenient experimentally, because it is reasonably soluble in water; and computationally, because it is quite rigid and lacks acidic or basic groups. 
In addition, it has attracted particular interest because of the high binding affinities of some guests, exceeding even the tightest-binding protein-ligand systems~\cite{liu_cucurbituril_2005, rekharsky_synthetic_2007, moghaddam_hostguest_2009, cao_attomolar_2014}.  
Finally, CB7 is already familiar to a number of computational chemistry groups, as it figured in two of the three SAMPL challenges that included host-guest components~\cite{muddana_sampl3_2012, muddana_sampl4_2014}, and it is currently the focus of the ``hydrophobe challenge''~\cite{schreiner_theoretical_2016}. 
 
\paragraph{CB7 presents several challenges} Despite the simplicity of CB7, calculations of its binding thermodynamics are still challenging, with several known complexities:

\begin{enumerate}
\item{{\bf Tight exit portal}: Guest molecules with bulky hydrophobic cores, such as adamantyl or [2.2.2]bicyclooctyl~\cite{moghaddam_hostguest_2009, moghaddam_new_2011} groups, do not fit easily through the constrictive portals~\cite{velez-vega_force_2012}.
As a consequence, free energy methods which compute the work of binding along a physical dissociation pathway may encounter a high barrier as the bulky core exits the cavity, and this can lead to subtle convergence problems~\cite{velez-vega_overcoming_2013, henriksen_computational_2015}. 
One way to solve this problem is to reversibly add restraints that open the portal, then remove the guest, and finally reversibly remove the restraints~\cite{henriksen_computational_2015}, including all of these contributions in the overall work of dissociation.}

\item{{\bf Water binding and unbinding}: If one computes the work of removing the guest from the host by a nonphysical pathway, in which the bound guest is gradually decoupled from the host and surrounding water~\cite{gilson_statistical-thermodynamic_1997}, large fluctuations in the number of water molecules within the host's cavity can occur when the guest is partly decoupled, and these fluctuations can slow convergence~\cite{rogers_role_2013}.}

\item{{\bf Salt concentration and buffer conditions}: Binding thermodynamics are sensitive to the composition of dissolved salts, both experimentally~\cite{moghaddam_new_2011, moghaddam_hostguest_2009, muddana_sampl4_2014} and computationally~\cite{muddana_blind_2014,hsiao_prediction_2014}. As a consequence, to be valid, a comparison of calculation with experiment must adequately model the experimental salt conditions.}

\item{{\bf Finite-size artifacts due to charge modification}: Because many guest molecules carry net charge, it should be ascertained that calculations in which guests are decoupled from the system do not generate artifacts related to the treatment of long-ranged Coulombic interactions~\cite{rocklin_calculating_2013, lin_overview_2014, reif_net_2014, simonson_concepts_2016}.}
%Re-order these based on generality? i.e. #1 is not that relevant to some.
\end{enumerate}

\paragraph{The proposed CB7 benchmark sets comprise two compound series}
For CB7, we have selected two sets of guests that were studied experimentally under uniform conditions (50 mM sodium acetate buffer, pH 4.74, 298K) by one research group ~\cite{liu_cucurbituril_2005, cao_attomolar_2014}. Each series is based on a common chemical
scaffold, making it amenable to not only absolute but also alchemical relative free energy calculations (Section~\ref{sec:FEMethods}). One set is based on an adamantane core ({\bf Table~\ref{cb7_benchmark1}}), and the other on an aromatic ring ({\bf Table~\ref{cb7_benchmark2}}).
These systems can be run to convergence to allow detailed comparisons among methods and with experiment.
Their binding free energies range from -5.99 to -17.19 kcal/mol, with the adamantane series spanning a particularly large range of free energies. 


\paragraph{Prior studies provide additional insight into CB7's challenges}
%Sampling
Sampling of the host appears relatively straightforward in CB7 as it is quite rigid and its symmetry provides for clever convergence checks~\cite{henriksen_computational_2015, monroe_converging_2014}. 
Due to its top-bottom symmetry, flips of guests from ``head-in'' to ``head-out'' configurations are not necessary to obtain convergence~\cite{fenley_bridging_2014}.
However, sampling of the guest geometry can be a challenge, with transitions between binding modes as slow as 0.07 flips/ns~\cite{monroe_converging_2014}, and flexible guests also presenting challenges~\cite{monroe_converging_2014}. 
As noted above, water sampling can also be an issue, with wetting/dewetting transitions occurring on the 50 ns timescale~\cite{rogers_role_2013}.

%System
Salt and buffer conditions are also key. 
In addition to the strong salt-dependence of binding~\cite{moghaddam_new_2011}, acetic acid (such as in a sodium acetate buffer) can compete with guests for the binding site~\cite{moghaddam_hostguest_2009}. 
This may partially explain systematic errors in some computational studies~\cite{muddana_blind_2014,hsiao_prediction_2014}. 
Indeed, the difference between 50 mM sodium acetate buffer and 100 mM sodium phosphate buffer impacts measured binding free energies by 2.5-2.8 kcal/mol~\cite{muddana_blind_2014, muddana_sampl4_2014}.  
Cationic guests could also have substantial and differing interactions with the counterions in solution as well, potentially lowering affinity relative to zero-salt conditions~\cite{muddana_sampl4_2014}.  
Thus, one group found a 6.4-6.8 kcal/mol dependence on salt concentration~\cite{hsiao_prediction_2014}, possibly impacting other studies as well~\cite{monroe_converging_2014}

%Force field
Despite these issues, CB7 appears to be at the point where careful studies can probe the true accuracy of our force fields~\cite{henriksen_computational_2015, gao_binding_2015, yin_toward_2015}, and the results can be sobering, with RMS errors in the binding free energies as high as 8 kcal/mol~\cite{henriksen_computational_2015, monroe_converging_2014}. More encouragingly, the values of $R^2$ values can be as high as 0.92~\cite{henriksen_computational_2015}. 
Some force fields appear relatively worse than others~\cite{hsiao_prediction_2014, muddana_prediction_2012}.
Calculated values are in many cases quite sensitive to details of force field parameters~\cite{monroe_converging_2014, moghaddam_new_2011, muddana_prediction_2012}. For example, modest modification of some Lennard-Jones parameters yielded dramatic improvements in calculated values~\cite{yin_toward_2015}, and host-guest binding data has, accordingly, been suggested as an input for force field development~\cite{yin_toward_2015, henriksen_computational_2015, gao_binding_2015}.
Water structure around CB7 and calculated binding enthalpies also appear particularly sensitive to the choice of water model~\cite{rogers_role_2013, fenley_bridging_2014, gao_binding_2015}, and water is clearly important for modulating binding~\cite{nguyen_grid_2012}.
The water model also impacts the number of sodium ions which must be displaced (in sodium-based buffer) on binding~\cite{gao_binding_2015, henriksen_computational_2015}.

Despite its apparent simplicity, CB7 is still a challenging benchmark that can put important issues into high relief. For example, in SAMPL4, free energy methods yielded $R^2$ values from 0.1 to 0.8 and RMS errors of about 1.9 to 4.9 kcal/mol for the same set of CB7 cases. This spread of results across rather similar methods highlights the need for shared benchmarks. Potential explanations include convergence difficulties, subtle methodological differences, and details of how the methods were applied~\cite{muddana_sampl4_2014}. Until the origin of such discrepancies is clear, it is difficult to know how accurate our methods truly are. 

\begingroup
\squeezetable
\begin{table*}
%\tabcolsep7pt
\caption{Proposed CB7 Set 1 benchmark data}
\label{cb7_benchmark1}
\begin{tabular}{lllp{1.5cm}>{\ttfamily}lS[table-format=-1.2, table-figures-uncertainty=1]l}
\toprule
\multicolumn{1}{c}{ID$^{\rm a}$} & \multicolumn{1}{c}{name} & \multicolumn{1}{c}{PC CID$^{\rm b}$} & \multicolumn{1}{c}{2D} & \multicolumn{1}{c}{\rmfamily SMILES} & \multicolumn{1}{c}{$\Delta G^{\rm c}$ \,{(}kcal/mol{)}} & \\
\midrule

1  & memantine                          & 4054      & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/4054.pdf}}      & CC12CC3CC(C1)(CC(C3)(C2)N)C                  & -5.99 \pm 0.05  & \kern-1em {$^{\rm d}$}  \\
3  & 1,3-bis(trimethylaminio)adamantane	& 101379195 & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/101379195.pdf}} & C[N+](C)(C)C12CC3CC(C1)CC(C3)(C2)[N+](C)(C)C & -6.55 \pm 0.05  & \kern-1em {$^{\rm d}$}  \\
5  & N-(1-adamantyl)ethylenediamine     & 303798    & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/303798.pdf}}    & C1C2CC3CC1CC(C2)(C3)NCCN                     & -18.22 \pm 0.09 & \kern-1em {$^{\rm e}$}  \\
17 & adamantane-1,3-diamine             & 213512    & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/213512.pdf}}    & C1C2CC3(CC1CC(C2)(C3)N)N                     & -11.33 \pm 0.05 & \kern-1em {$^{\rm d}$}  \\
18 & 1-adamantanecarboxylic acid        & 13235     & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/13235.pdf}}     & C1C2CC3CC1CC(C2)(C3)C(=O)O                   & -11.59 \pm 0.06 & \kern-1em {$^{\rm d}$}  \\
22 & 1-Adamantyltrimethylaminium        & 3010127   & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/3010127.pdf}}   & C[N+](C)(C)C12CC3CC(C1)CC(C3)C2              & -16.66 \pm 0.08 & \kern-1em {$^{\rm d}$}  \\
23 & amantadine                         & 2130      & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/2130.pdf}}      & C1C2CC3CC1CC(C2)(C3)N                        & -17.19 \pm 0.08 & \kern-1em {$^{\rm d}$}  \\
24 & N-(1-adamantyl)pyridinium          & 3848257   & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/3848257.pdf}}   & C1C2CC3CC1CC(C2)(C3)[N+]4=CC=CC=C4           & -16.75 \pm 0.07 & \kern-1em {$^{\rm d}$}  \\

\bottomrule
\end{tabular}\\

$^{\rm a}$ Compound ID from original paper; $^{\rm b}$ PubChem Compound ID; $^{\rm c}$ Standard binding free energy, where all measurements were done via NMR in 50mM sodium acetate buffer in $D_2O$ at pH 4.74 and 298 K. Uncertainties are obtained by taking the reported standard deviations across triplicate measurements~\cite{isaacs_personal_2016} and dividing by $\sqrt 3$;  $^{\rm d}$ drawn from~\cite{liu_cucurbituril_2005}; $^{\rm e}$ drawn from ~\cite{cao_attomolar_2014}.
\end{table*}
\endgroup

\begingroup
\squeezetable
\begin{table*}
\tabcolsep7.5pt
\caption{Proposed CB7 Set 2 benchmark data}
\label{cb7_benchmark2}
\begin{tabular}{lllp{2cm}>{\ttfamily}lS[table-format=-1.2, table-figures-uncertainty=1]}
\toprule
\multicolumn{1}{c}{ID$^{\rm a}$} & \multicolumn{1}{c}{name} & \multicolumn{1}{c}{PC CID$^{\rm b}$} & \multicolumn{1}{c}{2D}  & \multicolumn{1}{c}{\rmfamily SMILES} & \multicolumn{1}{c}{$\Delta G^{\rm c,d}$ \,{(}kcal/mol{)}} \\
\midrule
2   & dopamine                 & 681       & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/681.pdf}}   & C1=CC(=C(C=C1CCN)O)O                 & -6.31 \pm 0.05                                            \\
4   & o-phenylenediamine       & 7243      & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/7243.pdf}}  & C1=CC=C(C(=C1)N)N                    & -6.68 \pm 0.05                                            \\
5   & m-phenylenediamine       & 7935      & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/7935.pdf}}  & C1=CC(=CC(=C1)N)N                    & -6.69 \pm 0.02                                            \\
7   & 4-(aminomethyl)pyridine  & 77317     & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/77317.pdf}} & C1=CN=CC=C1CN                        & -7.56 \pm 0.06                                            \\
8   & p-phenylenediamine       & 7814      & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/7814.pdf}}  & C1=CC(=CC=C1N)N                      & -8.60 \pm 0.06                                            \\
9   & p-toluidine              & 7813      & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/7813.pdf}}  & CC1=CC=C(C=C1)N                      & -9.43 \pm 0.05                                            \\
20  & p-xylylenediamine        & 68315     & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/68315.pdf}} & C1=CC(=CC=C1CN)CN                    & -12.62 \pm 0.06                                           \\
\bottomrule
\end{tabular}                                                                                                                                                                                                                                                           \\
$^{\rm a}$ Compound ID from original paper; $^{\rm b}$ PubChem Compound ID; $^{\rm c}$ Standard binding free energy, where all measurements were done via NMR in 50mM sodium acetate buffer in $D_2O$ at pH 4.74 and 298 K. Uncertainties are obtained by taking the reported standard deviations across triplicate measurements~\cite{isaacs_personal_2016} and dividing by $\sqrt 3$;  $^{\rm d}$ drawn from~\cite{liu_cucurbituril_2005}.
\end{table*}
\endgroup


\subsubsection{Gibb Deep Cavity Cavitands (GDCC)}


The octa-acids (OA) ({\bf Figure~\ref{hosts}}) are synthetic hosts with deep, basket-shaped, hydrophobic binding sites~\cite{gibb_well-defined_2004}. 
The eight carboxylic acidic groups for which they were originally named make these hosts water-soluble, but do not interact directly with bound hosts; instead, they project outward into solvent. 
Binding data have been reported for the original form of this host (OA)~\cite{gibb_well-defined_2004} and for a derivative with four added methyl groups at equivalent locations in the entryway, where they can contact a bound guest (TEMOA)~\cite{gan_nonmonotonic_2011, sullivan_binding_2016}. 
(Note that OA and TEMOA have also been called OAH and OAMe, respectively~\cite{yin_overview_2016}.) 
Additional family members with other substituents around the portal have been reported, as has a new series in which the eponymic carboxylic groups are replaced by various other groups, including a number of basic amines~\cite{hillyer_synthesis_2016}. 
However, we are not aware of binding data for these derivatives. 
In view of these other hosts, however, we propose the more general name Gibb deep cavity cavitands (GDCCs) for this family of hosts. 
The binding cavities of the GDCCs are fairly rigid, though less so than the cucurbiturils. 
Some simulators report ``breathing'' motions that vary the diameter of the entry by up to 8~\AA \cite{mikulskis_free-energy_2014}; and, in some studies, the benzoic acid ``flaps'' around the entry occasionally flip upward and into contact with the guest~\cite{yin_sampl5_2016, tofoleanu_absolute_2016}, though this motion has not been verified experimentally. 
Additionally, the four priopionate groups protruding into solution from the exterior base of the cavity are all flexible. 

The octa-acids tend to bind guest molecules possessing a hydrophobic moiety that fits into the host's cavity and a hydrophilic moiety that projects into the aqueous solvent.  
Within these specifications, they bind a diversity of ligands, including both organic cations and anions, as well as neutral compounds with varying degrees of polarity~\cite{gibb_guests_2009, gibb_binding_2013}.  
Compounds with adamantane or noradamantane groups display perhaps the highest affinities observed so far, with binding free energies ranging to about -8 kcal/mol~\cite{sun_calorimetric_2008}.  
Much of the experimental binding data comes from ITC, so binding enthalpies are often available. 

Two experimental aspects of binding are particularly intriguing and noteworthy. 
First, the binding thermodynamics of OA is sensitive to the type and concentration of anions in solution. 
Although NaCl produces relatively modest effects, 100 mM sodium perchlorate, chlorate and isothiocyanate can shift binding enthalpies by up to about 10 kcal/mol and free energies by around 2 kcal/mol~\cite{gibb_anion_2011}. 
These effects are due in part to binding of anions by the host; indeed, trichloroacetate is reported to bind OA with a free energy of -5.2 kcal/mol~\cite{sokkalingam_binding_2016}, and competition of other guests with bound anions leads to entropy-enthalpy tradeoffs. 
Second, elongated guests can generate ternary complexes, in which two OA hosts encapsulate one guest, especially if both ends of the guest are not very polar~\cite{gibb_guests_2009}.  

\paragraph{The proposed GDCC benchmark sets are drawn from SAMPL}
As a core benchmark series for this family, we propose two sets which formed part of the SAMPL4 and SAMPL5 challenges, based on adamantane derivatives (Table~\ref{gdcc_benchmark1}) and cyclic (aromatic and saturated) carboxylic acids (Table~\ref{gdcc_benchmark2}) binding to hosts OA and TEMOA with free energies of -3.7 to -7.6 kcal/mol. These cases offer aqueous binding data with a reasonably broad range of binding free energies, frequently along with binding enthalpies; the hosts and many or all of their guests are small and rigid enough to allow convincing convergence of binding thermodynamics with readily feasible simulations; and, like the cucurbiturils, they are already emerging as \emph{de facto} computational benchmarks, due to their use in the SAMPL4 and SAMPL5 challenges~\cite{muddana_sampl4_2014, yin_overview_2016}. 

\paragraph{OA introduces new challenges beyond CB7} Issues deserving attention when interpreting the experimental data and calculating the binding thermodynamics of these systems include the following:

\begin{enumerate}
\item{{\bf Tight exit portal}: The methyl groups of the TEMOA variant narrow the entryway and can generate a barrier to the entry or exit of guest molecules with bulky hydrophobic cores, though the degree of constriction is not as marked as for CB7 (above). 
The TEMOA methyls groups can additionally hinder sampling of guest poses in the bound state, leading to convergence problems~\cite{yin_overview_2016} specific to TEMOA. }

\item{{\bf Host conformational sampling}: Although the flexible propionate groups are not proximal to the binding cavity, they are charged and so can have long-ranged interactions. 
As a consequence, it may be important to ensure their conformations are well sampled, though motions may be slow~\cite{mikulskis_free-energy_2014}. 
Similarly, benzoic acid flips~\cite{yin_sampl5_2016, tofoleanu_absolute_2016} could potentially be an important challenge in some force fields.}

\item{{\bf Water binding and unbinding:} Water appears to undergo slow motions into and out of the OA host, on timescales upwards of 5 ns~\cite{ewell_water_2008}. 
This poses significant challenges for some approaches, such as metadynamics, where deliberately restraining water to stay out of the cavity when the host is not bound (and computing the free energy of doing so) can help convergence~\cite{bhakat_resolving_2016}, and perhaps for other methods as well.}

\item{{\bf Salt concentration and buffer conditions:} As in the case of CB7, binding to GDCCs is modulated by the composition of dissolved salts, both experimentally~\cite{gibb_anion_2011, sokkalingam_binding_2016} and computationally~\cite{pal_combined_2016, tofoleanu_absolute_2016}. As a consequence, to be valid, a comparison of calculation with experiment must adequately model the experimental salt conditions.
}

\item{{\bf Finite-size artifacts due to charge modification}: As for CB7, it should be ascertained that calculations in which charged guests are decoupled from the system do not generate artifacts related to long range Coulomb interactions.~\cite{rocklin_calculating_2013, lin_overview_2014, reif_net_2014, simonson_concepts_2016}.}

\item{{\bf Protonation state effects}: Although experiments are typically run at pH values that lead to well-defined protonation states of the host and its guests, this may not always hold~\cite{muddana_sampl4_2014, ewell_water_2008, tofoleanu_absolute_2016}, particularly given experimental evidence for extreme binding-driven pKa shifts of 3-4 log units for some carboxylate compounds~\cite{wang_itc_2016, sokkalingam_binding_2016}. 
Thus, attention should be given to ionization states and their modulation by binding.}
\end{enumerate}

\begingroup
\squeezetable
\begin{table*}
% \tabcolsep7.5pt
\caption{Proposed GDCC Set 1 benchmark data}
\label{gdcc_benchmark1}
\begin{tabular}{lllp{1.5cm}>{\ttfamily}lS[table-format=-1.3, table-figures-uncertainty=1]S[table-format=-1.2, table-figures-uncertainty=1]}
\toprule
\multicolumn{1}{c}{ID$^{\rm a}$} & \multicolumn{1}{c}{name} & \multicolumn{1}{c}{PC CID$^{\rm b}$}  & \multicolumn{1}{c}{2D} & \multicolumn{1}{c}{\rmfamily SMILES} & \multicolumn{1}{c}{$\Delta G^{\rm c}$ \,{(}kcal/mol{)}} & \multicolumn{1}{c}{$\Delta H^{\rm d}$ \,{(}kcal/mol{)}} \\
\midrule
\multicolumn{7}{c}{Octa Acid binders} \\ 
\midrule                                                                                                                                                        
3 / OA-G1    & 5-hexynoic acid                     & 143036           & \parbox[c]{1em}{\includegraphics[scale=0.15]{figures/143036.pdf}}   & C\#CCCCC(=O)O                      & -5.40 \pm 0.003    & -7.71 \pm 0.05     \\
4 / OA-G6    & 3-nitrobenzoic acid                 & 8497             & \parbox[c]{1em}{\includegraphics[scale=0.15]{figures/8497.pdf}}     & C1=CC(=CC(=C1)[N+](=O)[O-])C(=O)O  & -5.34 \pm 0.005    & -5.67 \pm 0.01     \\
5 / OA-G2    & 4-cyanobenzoic acid                 & 12087            & \parbox[c]{1em}{\includegraphics[scale=0.15]{figures/12087.pdf}}    & C1=CC(=CC=C1C\#N)C(=O)O            & -4.73 \pm 0.01     & -4.45 \pm 0.08     \\
6 / OA-G4    & 4-bromoadamantane-1-carboxylic acid & 12598766         & \parbox[c]{1em}{\includegraphics[scale=0.15]{figures/12598766.pdf}} & C1C2CC3CC(C2)(CC1C3Br)C(=O)O       & -9.37 \pm 0.01     & -14.78 \pm 0.02    \\  
7 / OA-G3    & N,N,N-trimethylhexan-1-aminium      & 84774            & \parbox[c]{1em}{\includegraphics[scale=0.15]{figures/84774.pdf}}    & CCCCCC[N+](C)(C)C                  & -4.49 \pm 0.01     & -5.91 \pm 0.10     \\ 
8 / OA-G5    & trimethylphenethylaminium           & 14108            & \parbox[c]{1em}{\includegraphics[scale=0.15]{figures/14108.pdf}}    & C[N+](C)(C)CCC1=CC=CC=C1           & -3.72 \pm 0.01     & -9.96 \pm 0.11     \\
\midrule
\multicolumn{7}{c}{TEMOA/OAMe binders} \\ 
\midrule
3 / OA-G1    & 5-hexynoic acid                     & 143036           & \parbox[c]{1em}{\includegraphics[scale=0.15]{figures/143036.pdf}}   & C\#CCCCC(=O)O                      & -5.476 \pm 0.006   & -9.961 \pm 0.006   \\
4 / OA-G6    & 3-nitrobenzoic acid                 & 8497             & \parbox[c]{1em}{\includegraphics[scale=0.15]{figures/8497.pdf}}     & C1=CC(=CC(=C1)[N+](=O)[O-])C(=O)O  & -4.52 \pm 0.02     & -9.1 \pm 0.1       \\
5 / OA-G2    & 4-cyanobenzoic acid                 & 12087            & \parbox[c]{1em}{\includegraphics[scale=0.15]{figures/12087.pdf}}    & C1=CC(=CC=C1C\#N)C(=O)O            & -5.26 \pm 0.01     & -7.6 \pm 0.1       \\
6 / OA-G4    & 4-bromoadamantane-1-carboxylic acid & 12598766         & \parbox[c]{1em}{\includegraphics[scale=0.15]{figures/12598766.pdf}} & C1C2CC3CC(C2)(CC1C3Br)C(=O)O       & {ND$^{\rm e}$}       & {ND$^{\rm e}$}      \\  
7 / OA-G3    & N,N,N-trimethylhexan-1-aminium      & 84774            & \parbox[c]{1em}{\includegraphics[scale=0.15]{figures/84774.pdf}}    & CCCCCC[N+](C)(C)C                  & -5.73 \pm 0.06     & -6.62 \pm 0.2      \\ 
8 / OA-G5    & trimethylphenethylaminium           & 14108            & \parbox[c]{1em}{\includegraphics[scale=0.15]{figures/14108.pdf}}    & C[N+](C)(C)CCC1=CC=CC=C1           & {ND$^{\rm e}$}       & {ND$^{\rm e}$}      \\
\bottomrule
\end{tabular}\\
$^{\rm a}$ Compound ID from ~\cite{sullivan_binding_2016} and SAMPL5 ID from ~\cite{yin_overview_2016}; $^{\rm b}$ PubChem Compound ID; $^{\rm c}$ Standard binding free energy from~\cite{sullivan_binding_2016}, where all measurements were done via ITC in 50 mM sodium phosphate buffer at pH 11.5 and 298 K. Uncertainties, drawn from the experimental paper, were computed from triplicate measurements taken with freshly made solutions of host and guest. However, based on personal communication with the authors, it may be advisable to regard the accuracy more conservatively, at $\sim$2\% for $\Delta$G and $\sim$6\% for $\Delta$H;  $^{\rm d}$ measured binding enthalpy~\cite{sullivan_binding_2016}, subject to the same conditions/caveats as $^{\rm c}$. $^{\rm e}$ not done.
\end{table*}
\endgroup

\begingroup
\squeezetable
\begin{table*}
% \tabcolsep7.5pt
\caption{Proposed GDCC Set 2 benchmark data}
\label{gdcc_benchmark2}
\begin{tabular}{lllp{2cm}>{\ttfamily}llS[table-format=-1.2, table-figures-uncertainty=1]}
\toprule
\multicolumn{1}{c}{ID$^{\rm a}$} & \multicolumn{1}{c}{name} & \multicolumn{1}{c}{PC CID$^{\rm b}$} & \multicolumn{1}{c}{2D}  & \multicolumn{1}{c}{\rmfamily SMILES} & \multicolumn{1}{c}{Method} & \multicolumn{1}{c}{$\Delta G^{\rm c}$ \,{(}kcal/mol{)}} \\
\midrule
1            & benzoic acid                             & 243              & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/243.pdf}}   & C1=CC=C(C=C1)C(=O)O              & NMR    & -3.72 \pm 0.03     \\
2            & 4-methylbenzoic acid                     & 7470             & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/7470.pdf}}  & CC1=CC=C(C=C1)C(=O)O             & NMR    & -5.85 \pm 0.06     \\
3            & 4-ethylbenzoic acid                      & 12086            & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/12086.pdf}} & CCC1=CC=C(C=C1)C(=O)O            & ITC    & -6.27 \pm 0.01     \\
4            & 4-chlorobenzoic acid                     & 6318             & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/6318.pdf}}  & C1=CC(=CC=C1C(=O)O)Cl            & ITC    & -6.72 \pm 0.01     \\
5            & 3-chlorobenzoic acid                     & 447              & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/447.pdf}}   & C1=CC(=CC(=C1)Cl)C(=O)O          & NMR    &  -5.24 \pm 0.02    \\
6            & cyclohexanecarboxylic acid               & 7413             & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/7413.pdf}}  & C1CCC(CC1)C(=O)O                 & NMR    & -5.62 \pm 0.04     \\
7            & trans-4-methylcyclohexanecarboxylic acid & 20330            & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/20330.pdf}} & [C@@H]1(CC[C@@H](CC1)C(=O)O[H])C & ITC    & -7.61 \pm 0.04     \\
\bottomrule
\end{tabular}\\
$^{\rm a}$ Compound ID from original paper ~\cite{gibb_binding_2013}; $^{\rm b}$ PubChem Compound ID; $^{\rm c}$ Standard binding free energy from ~\cite{gibb_binding_2013}, where all measurements were done in 10 mM sodium tetraborate buffer at pH 9.2 and 298 K. A quirk is that for the NMR measurements, the guest was titrated in from 50 mM sodium tetraborate buffer, so the buffer concentration changed during the titration. Uncertainty is the standard error of the mean in free energy, computed from the reported standard deviations in $K_a$. Again, based on personal communication with the authors, uncertainties of perhaps 10\% may be more appropriate.
\end{table*}
\endgroup

\paragraph{Prior studies provide additional insight into the challenges of OA}

%SAMPLING
As noted, two different host conformational sampling issues have been observed, with dihedral transitions for the proprionate groups occurring on 1-2 ns timescales~\cite{mikulskis_free-energy_2014}); motions of the benzoic acid flaps were also relatively slow~\cite{yin_sampl5_2016, tofoleanu_absolute_2016} though perhaps thermodynamically unimportant. 
Guest sampling can also be an issue, at least in TEMOA~\cite{yin_overview_2016}, and this hosts's tight cavity may also have implications for binding entropy~\cite{yin_sampl5_2016}.

% SYSTEM:
Salt concentration strongly modulates binding affinity, at least for anions, and the nature of the salt also plays an important role~\cite{carnegie_anion_2014}. 
Co-solvent anions can also increase \emph{or} decrease binding depending on their identity~\cite{gibb_anion_2011}. 
Some salts even bind to OA themselves, with perchlorate~\cite{gibb_anion_2011} and trichloroacetate~\cite{sokkalingam_binding_2016} being particularly potent, and thus will compete with guests for binding.  
Computationally, including additional salt beyond that needed for system neutralization changed binding free energies by up to 4 kcal/mol~\cite{tofoleanu_absolute_2016}.  

Naively, protonation states of the guests might seem clear and unambiguous.
But since OA can bind guests of diverse net charges, the protonation state may not always be clear.  
One study used absolute binding free energy calculations for different guest charge states, coupled with pKa calculations, and found that inclusion of pKa corrections and the possibility of alternate charge states of the guests affected calculated binding free energies by up to 2 kcal/mol~\cite{tofoleanu_absolute_2016}. As noted above, experimental evidence also indicates major pKa shifts on binding so that species such as acetate, formate and others would bind in neutral form at neutral pH~\cite{wang_itc_2016, sokkalingam_binding_2016}. 
Even the host protonation state may be unclear; while OA is often assumed to have all eight carboxylic acids deprotonated at the basic pH of typical experiments, the four at the bottom are in close proximity, and these might make hydrogen bonds allowing retention of two protons~\cite{ewell_water_2008}.  
Thus, there are uncertainties as to the host protonation state~\cite{muddana_sampl4_2014, ewell_water_2008}, which perhaps also could be modulated by guest binding.  

%FORCE FIELD, etc.:
%The challenges of system definition and conformational sampling appear to be greater for OA and TEMOA than for CB7, , so evidence of force field limitations, if present, is not yet as clear. There have been some comparisons of charge~\cite{mikulskis_free-energy_2014, muddana_sampl4_2014, monroe_converging_2014} and water models~\cite{yin_sampl5_2016}, but differences so far seem mostly inconclusive or at least not that large. Similarly, OPLS and GAFF results did not appear dramatically different in accuracy~\cite{bhakat_resolving_2016} 
%The above paragraph could be cut for space if we need

Several groups used different methods but the same force field and water model in SAMPL5, with rather varied levels of success because of discrepancies in calculated free energies~\cite{yin_overview_2016, bosisio_blinded_2016, bhakat_resolving_2016}. 
However, some of these issues were resolved in follow-up work~\cite{bhakat_resolving_2016}, bringing the methods into fairly good agreement for the majority of cases~\cite{yin_sampl5_2016, bosisio_blinded_2016}.

\subsection{Protein-ligand benchmarks: the T4 lysozyme model binding sites}

\begin{figure*}
\includegraphics[width=\textwidth]{figures/lysozyme.pdf}
\caption{\label{fig:lysozyme} Benzene and hexylbenzene in the lysozyme L99A site, and phenol and 4,5,6,7-tetrahydroindole in the L99A/M102Q site (PDBs 4W52, 4W59, 1LI2, and 3HUA, respectively). The binding site shape is shown as a semi-transparent surface, and the protein shown with cartoons. In both cases, the structure with the smaller ligand is shown in green and that with the larger ligand is shown in blue, and the larger ligand induces a motion of helix F bordering the binding site. Phenol and 4,5,6,7-tetrahydroindole both also bind with an ordered water, though this does not occur for all ligands in the polar L99A/M102Q site.}
\end{figure*}


Although we seek ultimately to predict binding in systems of direct pharmaceutical relevance, simpler protein-ligand systems can represent important stepping stones in this direction.
Two model binding sites in T4 lysozyme have been particularly useful in this regard ({\bf Figure~\ref{fig:lysozyme}}). 
These two binding sites, called L99A~\cite{morton_energetic_1995, morton_specificity_1995} and L99A/M102Q~\cite{wei_model_2002, graves_decoys_2005} for point mutations which create the cavities of interest, are created in artificial mutants of phage T4 lysozyme, and have been studied extensively experimentally and via modeling. 
As protein-ligand systems, they introduce additional complexities beyond those observed in host-guest systems, yet they share some of the same simplicity.
The ligands are generally small, neutral, and relatively rigid, with clear protonation states.
In many cases, substantial protein motions are absent, allowing calculated binding free energies to apparently converge relatively easily. 
However, like host-guest systems, these binding sites are still surprisingly challenging~\cite{mobley_use_2006, mobley_confine_2007, mobley_predicting_2007, boyce_predicting_2009, jiang_free_2010, gallicchio_binding_2010, lim_sensitivity_2016}. In addition, precise convergence is sometimes difficult to achieve, and it is in all cases essentially impossible to fully verify.  As a consequence, these are ``soft benchmarks" as defined above (Section~\ref{subsec:benchmarktypes}).
The importance of the lysozyme model sites is also driven by the relative wealth of experimental data. 
It is relatively easy to identify new ligands and obtain high quality crystal structures and affinity measurements, allowing two different rounds of blind predictions testing free energy calculations~\cite{mobley_predicting_2007, boyce_predicting_2009}. 


\subsubsection{The apolar and polar cavities and their ligands}
The L99A site is also called the ``apolar'' cavity. It is relatively flat and elongated, and binds mostly nonpolar molecules such as benzene, toluene, p-xylene, and n-butylbenzene: basically, a fairly broad range of nonpolar planar five- and six-membered rings and ring systems (such as indole).
The polar version, L99A/M102Q, introduces an additional point mutation along one edge of the binding site, providing a glutamine that introduces polarity and the potential for hydrogen bonding. 
It still binds a variety of nonpolar ligands such as toluene (though not benzene). 
One small downside of these binding sites is that the range of affinities is relatively narrow: about -4.5 to -6.7 kcal/mol in the apolar site~\cite{morton_energetic_1995, mobley_predicting_2007}, and about -4 to -5.5 kcal/mol in the polar site~\cite{boyce_predicting_2009}. Thus, even the strongest binders are not particularly strong, and the weakest binders tend to run up against their solubility limits. 
Still, these sites offer immensely useful tests for free energy calculations.

For both sites, fixed charge force fields seem to yield reasonably accurate free energies, with RMS errors between 1-2 kcal/mol, and some level of correlation with experiment, despite limited dynamic range~\cite{deng_calculation_2006, mobley_predicting_2007, boyce_predicting_2009, gallicchio_binding_2010, wang_identifying_2013}. 
System composition/preparation issues also do not seem to be a huge factor.
Instead, sampling issues predominate:
\begin{enumerate}

\item{Ligand binding mode/orientational sampling: The binding sites are buried and roughly oblong, with ligands which are similar in shape. 
Ligands with axial symmetry typically have at least two reasonably likely binding modes, but broken symmetry can drive up the number of likely binding modes. 
For example, phenol has two plausible binding modes in the polar cavity~\cite{graves_rescoring_2008, boyce_predicting_2009} but 3-chlorophenol has at least four, three of which appear to have some population in simulations~\cite{gallicchio_binding_2010}, because the chlorine could point in either direction within the site. 
Timescales for binding mode interconversion are relatively slow, with in-plane transitions on the 1-10 nanosecond timescale, and out-of-plane transitions (e.g. between toluene's two symmetry-equivalent binding modes) taking hundreds of nanoseconds (Mobley group, unpublished data).}

\item{Sidechain rearrangements: Some sidechains are known to reorganize when binding certain ligands. 
The smallest ligands tend not to induce conformational changes, but larger ligands may induce sidechain rearrangements -- often, rotamer flips -- around the binding site region. 
These can be slow in the tightly packed binding site. 
This especially occurs for Val111 in the L99A site~\cite{morton_specificity_1995, mobley_confine_2007, jiang_free_2010} and Leu118, Val11, and Val103 in L99A/M102Q~\cite{wei_model_2002, wei_testing_2004, graves_rescoring_2008, boyce_predicting_2009}. 
These sidechain motions typically present sampling problems for standard MD simulations~\cite{mobley_confine_2007, mobley_predicting_2007, boyce_predicting_2009, jiang_free_2010, wang_achieving_2012}.
}

\item{Backbone sampling: 
Larger ligands induce shifts of the F helix, residues 107 or 108 to 115, adjacent to the binding site, allowing the site to enlarge.
This occurs in both binding sites~\cite{wei_testing_2004, boyce_predicting_2009, merski_homologous_2015}, but is best characterized for L99A~\cite{merski_homologous_2015}.
There, addition of a series of methyl groups from benzene up to n-hexylbenzene causes a conformational transition in the protein from closed to intermediate to open conformations.
}
\end{enumerate}

{\bf Tables~\ref{apolar_benchmark} and~\ref{polar_benchmark}} introduce proposed benchmark sets for the apolar and polar cavities, giving ligands potentially amenable to both absolute and relative free energy calculations, and spanning the range of available affinities. 
Co-crystal structures are available in most cases, and the PDB IDs are provided in the tables. 
The selected ligands span a range of challenges and levels of difficulty, ranging from fairly simple to including most of the challenges noted above. 
Essentially all of them have been included in at least one prior computational study, and some have appeared in a variety of prior studies.
Additional known ligands and non-binders are available, with binding affinities available for 19 compounds in the L99A site~\cite{eriksson_cavity-containing_1992, morton_energetic_1995, mobley_predicting_2007} and 16 in L99A/M102Q~\cite{wei_model_2002, graves_decoys_2005, boyce_predicting_2009}. 
Because of the extent of the sampling challenges in lysozyme, binding of most ligands will currently constitute a soft benchmark, though long-timescale simulations to turn these into hard benchmarks may already be feasible.

\begingroup
\squeezetable
\begin{table*}
\tabcolsep7.5pt
\caption{Proposed Lysozyme L99A Set benchmark data}
\label{apolar_benchmark}
\begin{center}
\begin{tabular}{llp{2cm}>{\ttfamily}lS[table-format=-1.2, table-figures-uncertainty=1]ll}
\toprule
\multicolumn{1}{c}{name} & \multicolumn{1}{c}{PC CID$^{\rm h}$}  & \multicolumn{1}{c}{2D} & \multicolumn{1}{c}{\rmfamily SMILES} & \multicolumn{1}{c}{$\Delta G^{\rm a}$ \,{(}kcal/mol{)}}               & \multicolumn{1}{c}{PDB code} & \multicolumn{1}{c}{reference} \\
\midrule
benzene$^{\rm b}$                & 241                      & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/241.pdf}}   & c1ccccc1               & -5.19\pm0.16                       & 181L~\cite{morton_specificity_1995}, 4W52~\cite{merski_homologous_2015} & \cite{morton_energetic_1995}                                 \\   %PNAS series; 
toluene$^{\rm b}$                & 1140                     &  \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/1140.pdf}}  & Cc1ccccc1              & -5.52\pm0.04                       & 4W53~\cite{merski_homologous_2015}                                      & \cite{morton_energetic_1995}                                 \\   %PNAS series
ethylbenzene$^{\rm b}$           & 7500                     &  \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/7500.pdf}}  & CCc1ccccc1             & -5.76\pm0.07                       & 1NHB~\cite{morton_specificity_1995}, 4W54~\cite{merski_homologous_2015} & \cite{morton_energetic_1995}                                 \\ %PNAS series
propylbenzene$^{\rm b}$          & 7668                     &  \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/7668.pdf}}  & CCCc1ccccc1            & -6.55\pm0.02                       & 4W55~\cite{merski_homologous_2015}                                      & \cite{morton_energetic_1995}                                 \\ %#PNAS series
butylbenzene$^{\rm b}$           & 7705                     &  \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/7705.pdf}}  & CCCCc1ccccc1           & -6.70\pm0.02                       & 186L~\cite{morton_specificity_1995}, 4W57~\cite{merski_homologous_2015} & \cite{morton_energetic_1995}                                 \\%PNAS series
hexylbenzene$^{\rm b}$           & 14109                    &  \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/14109.pdf}} & CCCCCCc1ccccc1         & {UNK$^{\rm c}$}                       & 4W59~\cite{merski_homologous_2015}                                      & \cite{morton_energetic_1995}                                 \\%PNAS series
\emph{p}-xylene$^{\rm d}$        & 7809                     &  \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/7809.pdf}}  & Cc1ccc(cc1)C           & -4.67\pm0.06                       & 187L~\cite{morton_specificity_1995}                                     & \cite{morton_energetic_1995}                                 \\%Conformational change L99A
benzofuran                       & 9223                    &  \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/9223.pdf}}  & c1ccc2c(c1)cco2        & -5.46\pm0.03                       & 182L~\cite{morton_specificity_1995}                                     & \cite{morton_energetic_1995}                                 \\
thieno[2,3-c]pyridine            & 9224                    &  \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/9224.pdf}}  & c1cncc2c1ccs2          & {NB$^{\rm e}$}                         & ND$^{\rm f}$                                                            & \cite{mobley_predicting_2007}                                \\        %Isosteric with ligands
phenol$^{\rm g}$                 & 996                      &  \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/996.pdf}}   & c1ccc(cc1)O            & {NB$^{\rm e}$}                         & ND$^{\rm f}$                                                            & \cite{morton_energetic_1995, mobley_predicting_2007}         \\%Isosteric with ligands; binds in polar site (included for symmetry)

\hline
\end{tabular}\\
\end{center}
$^{\rm a}$T=302K, with compounds from \cite{morton_energetic_1995} measured in 50mM sodium acetate at pH 5.5 and thieno[2,3-c]pyridine measured at pH 6.8 in 50 mM potassium chloride and 38\% (v/v) ethylene glycol; $^{\rm b}$ part of the series of~\cite{merski_homologous_2015}, so larger ligands in the series induce conformational change; $^{\rm c}$ unknown due to solubility limitations, but likely binds strongly; $^{\rm d}$ L99A sidechain undergoes rotation; $^{\rm e}$ nonbinder; $^{\rm f}$ not done; $^{\rm g}$ included since it is a binder in the polar cavity; $^{\rm h}$ PubChem compound ID.
\end{table*}
\endgroup

\begingroup
\squeezetable
\begin{table*}
% \tabcolsep7.5pt
\caption{Proposed Lysozyme L99A/M102Q Set benchmark data}
\label{polar_benchmark}
\begin{center}
\begin{tabular}{llp{2cm}>{\ttfamily}lS[table-format=-1.2, table-figures-uncertainty=1]ll}
\toprule
\multicolumn{1}{c}{ligand}         & \multicolumn{1}{c}{PC CID$^{\rm k}$} & \multicolumn{1}{c}{2D}                                             & \multicolumn{1}{c}{\rmfamily SMILES} & \multicolumn{1}{c}{$\Delta G^{\rm a}$ \,{(}kcal/mol{)}} & \multicolumn{1}{c}{PDB code}      & \multicolumn{1}{c}{reference} \\
\midrule
toluene$^{\rm b}$                  & 1140                                 & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/1140.pdf}}     & Cc1ccccc1                            & -4.93                                                   & ND$^{\rm c}$                      & \cite{wei_model_2002}         \\ %included for symmetry with L99A site
phenol                             & 996                                & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/996.pdf}}      & c1ccc(cc1)O                          & -5.24                                                   & 1LI2~\cite{wei_model_2002}        & \cite{wei_model_2002}         \\ % relative set we did
catechol$^{\rm h}$                 & 289                                  & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/289.pdf}}      & c1ccc(c(c1)O)O                       & -4.16\pm0.03                                            & 1XEP~\cite{graves_decoys_2005}    & \cite{wei_model_2002}         \\ % relative set we did
2-ethoxyphenol$^{\rm d}$           & 66755                                & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/66755.pdf}}    & CCOc1ccccc1O                         & -4.02\pm0.03                                            & 3HU8~\cite{boyce_predicting_2009} & \cite{boyce_predicting_2009}  \\% relative set we did
benzyl acetate$^{\rm e, f}$        & 8785                                 & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/8785.pdf}}     & CC(=O)OCc1ccccc1                     & -4.48\pm0.16                                            & 3HUK~\cite{boyce_predicting_2009} & \cite{boyce_predicting_2009}  \\ %induces helix motions
4,5,6,7-tetrahydroindole$^{\rm f}$ & 57452536                             & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/57452536.pdf}} & c1c[nH]c2c1CCCC2                     & -4.61\pm0.09                                            & 3HUA~\cite{boyce_predicting_2009} & \cite{boyce_predicting_2009}  \\ % induces helix motion
n-phenylglycinonitrile$^{\rm g}$   & 76372                                & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/76372.pdf}}    & c1ccccc1NCC\#N                       & -5.52\pm0.18                                            & 2RBN~\cite{boyce_predicting_2009} & \cite{boyce_predicting_2009}  \\ 
3-chlorophenol                     & 7933                                 & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/7933.pdf}}     & c1cc(cc(c1)Cl)O                      & -5.51                                                   & 1LI3~\cite{wei_model_2002}        & \cite{wei_model_2002}         \\ 
2-methoxyphenol                    & 460                                  & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/460.pdf}}      & COc1ccccc1O                          & {NB$^{\rm i}$}                                          & ND$^{\rm c}$                      & \cite{boyce_predicting_2009}  \\ % relative set
4-vinylpyridine                    & 7502                                 & \parbox[c]{1em}{\includegraphics[scale=0.2]{figures/7502.pdf}}     & C=Cc1ccncc1                          & {NB$^{\rm i}$}                                          & ND$^{\rm c}$                      & \cite{wei_model_2002}         \\ 
\bottomrule
\end{tabular}                                                                                                                                                                                                                                                                                                       \\
\end{center}
$^{\rm a}$T=283K, with measurements done at pH 6.8 in 50 mM potassium phosphate, 200 mM potassium chloride buffer in the case of \cite{boyce_predicting_2009}; $^{\rm b}$ included for symmetry with the L99A site since this (unlike phenol and benzene) binds in both; $^{\rm c}$ not determined; $^{\rm d}$ fails to make crystallographic hydrogen bond~\cite{boyce_predicting_2009}; $^{\rm e}$ multiple binding modes; $^{\rm f}$ induces helix F motion; $^{\rm g}$ induces flip of Val111 sidechain; $^{\rm h}$ induces flip of Leu118 sidechain; $^{\rm j}$ nonbinder; $^{\rm k}$ PubChem compound ID.
\end{table*}
\endgroup


\subsubsection{Computational challenges posed by the T4 lysozyme benchmarks}
Early work on the lysozyme sites focused on the difficulty of predicting binding modes~\cite{mobley_use_2006, mobley_predicting_2007, boyce_predicting_2009} because of the slow interconversions noted above. 
Docking methods often can generate reasonable poses spanning most of the important possibilities~\cite{mobley_use_2006, mobley_predicting_2007, boyce_predicting_2009, graves_rescoring_2008} but do not accurately predict the binding mode of individual compounds~\cite{mobley_predicting_2007, boyce_predicting_2009, graves_rescoring_2008}. 
Thus it appears necessary to consider the possibility of multiple binding modes; this is also important since some ligands actually populate multiple binding modes~\cite{boyce_predicting_2009}. 
In a number of studies, candidate binding modes from docking are relaxed with MD simulations, then clustered to select binding modes for further study. 
It turns out an effective binding free energy for each distinct candidate binding mode can be computed separately~\cite{mobley_use_2006} and combined to find the population of each binding mode and determine the overall binding free energy.
However, this is costly since each candidate binding mode requires a full binding free energy calculation.

Relative binding free energy calculations do not dramatically simplify the situation. 
Introduction of a ligand modification can leave the binding mode uncertain (e.g., introducing a chlorine onto phenol leaves at least two possible binding modes even if the binding mode of phenol is known)~\cite{boyce_predicting_2009}. 
A na{\" i}ve solution is to consider multiple possible binding modes in relative free energy calculations~\cite{boyce_predicting_2009}, but this generates multiple results; determining the true relative binding free energy requires additional information~\cite{mobley_perspective_2012}. Enhanced sampling approaches provide one possible solution to the binding mode problem. 
Particularly, with $\lambda$ or Hamiltonian exchange techniques, ligands can easily switch between binding modes when they are non-interacting unless they are restrained, and then moves in $\lambda$ space can allow transitions back to the interacting state.
Thus, approaches employing this strategy can naturally sample multiple binding modes~\cite{gallicchio_binding_2010, wang_identifying_2013}.

While sidechain sampling has been a significant challenge, it is possible to use biased sampling techniques such as umbrella sampling to deliberately compute and include free energies of sampling slow sidechain rearrangements~\cite{mobley_confine_2007}. 
However, this is not a general solution, since it requires knowing what sidechains might rearrange on binding and then expending substantial computational power on sampling free energy landscapes for these rearrangements. 
An apparently better general strategy is including sidechains in enhanced sampling regions selected for Hamiltonian exchange~\cite{jiang_free_2010, khavrutskii_improved_2011} or REST~\cite{wang_achieving_2012}, allowing sidechains to be alchemically softened or torsion barriers lowered (or both), to enhance sampling at alchemical intermediate states.
With swaps between $\lambda$ values, enhanced sidechain sampling at intermediate states can propagate to all states, improving convergence~\cite{jiang_free_2010, wang_achieving_2012}.

Larger protein conformational changes in lysozyme have received less attention, partly because until very recently they seemed to be a peculiar oddity only rarely observed; i.e., for ligands 4,5,6,7-tetrahydroindole and benzyl acetate in the polar site~\cite{boyce_predicting_2009}. 
However, recent work noted above highlighted how a helix in the apolar cavity can open to accommodate larger ligands~\cite{merski_homologous_2015}. 
Timescales for this motion appear to be on the order of 50 ns, so it can pose sampling challenges, even for relative free energy calculations~\cite{lim_sensitivity_2016}. 
Including part of the protein in the enhanced sampling region via REST2 provides some benefits, but sampling these motions will likely prove a valuable test for enhanced sampling methods.

%This paragraph could be cut for space since it's speculative:
%Potentially, water sampling could also be a challenge in the polar binding site, as phenol binds with one ordered water~\cite{wei_model_2002} but to our knowledge, timescales for water sampling have not yet been examined, other than noting that water does not enter the polar binding site as ligands are removed~\cite{boyce_predicting_2009}. The apolar cavity is dry both with ligands present and with empty, so water sampling in the there cavity seems unlikely to be a challenge.

\section{BENCHMARK SYSTEMS FOR THE FUTURE}

While host-guest systems and lysozyme present an interesting range of challenges, we need additional benchmark systems that introduce a broader range of challenges and ultimately help us transition to applying these techniques with success to biological protein-ligand binding.
Several other systems have already received substantial attention in the literature and may be suited as additional near-term benchmark systems.

\subsection{CCP model sites}

Two model binding sites in cytochrome C peroxidase (CCP) created by the W191G mutant (the ``closed'' site~\cite{fitzgerald_ligand-gated_1996, rocklin_blind_2013} and then supplemented by an additional partial loop deletion (creating the ``open'' site~\cite{musah_artificial_2002, rosenfeld_excision_2002, rocklin_blind_2013}) have many similarities to the lysozyme binding sites. 
These, too, bind simple, fragment-like ligands in a buried cavity yet pose surprising challenges for simulations. Like lysozyme, discovery of new ligands is relatively straightforward~\cite{brenk_probing_2006, rocklin_blind_2013} and new crystal structures can be obtained fairly readily, driving use for blind tests of free energy calculations~\cite{rocklin_blind_2013}. 
Like lysozyme, binding free energies are mostly relatively weak (roughly -3.4 to -6.6 kcal/mol in the open site~\cite{rocklin_blind_2013}; however, ligands now include charged compounds, and the binding site contains a buried, charged ASP which makes strong electrostatic interactions with the ligands. 
Despite this strong interaction, however, some polar but neutral compounds still bind.

These binding sites appear to introduce new computational challenges as well. Particularly, work with two different force fields and rather different computational methods has found that alchemical techniques overestimate the range of affinities, by roughly a factor of three, in the closed site~\cite{banba_free_2000, banba_efficient_2000} and the same seems to hold true in the open site~\cite{rocklin_blind_2013}. 
There are some suggestions and evidence that this may be a result of force field artifacts -- particularly, a lack of screening of strong charge-charge interactions in a buried, otherwise relatively nonpolar environment~\cite{rocklin_blind_2013}. 
In addition to these potential force field challenges, the fact that ligands vary in formal charge also poses methodological challenges~\cite{rocklin_calculating_2013}. 
Since this system seems challenging yet tractable, and has been the focus of several free energy studies already may mean it is a good near-term benchmark for free energy methods.

\subsection{Thrombin}

Thrombin is a serine protease which is interesting both as a target in the coagulation cascade and as a representative serine protease. 
Because of its relevance as a pharmaceutical target, a wide variety of ligands and structures are available, driving its use in several prior free energy studies~\cite{wang_achieving_2012, schrodinger_accurate_2015, calabro_elucidation_2016}. 
Some of the available experimental binding data is of particularly high quality, from isothermal titration calorimetry, and highlights interesting trends such as non-additivity (in the form of positive coupling) between functional groups at different locations within two compound series~\cite{baum_non-additivity_2010, calabro_elucidation_2016}. 
Relative free energy calculations capture the experimental trends and, qualitatively, the non-additivity for these series (and achieve an overall mean unsigned error of 0.74 kcal/mol over a range of roughly 5 kcal/mol), though they do better at one series than the other~\cite{calabro_elucidation_2016}. 
This study suggests thrombin may be at the ``sweet spot'' for benchmark systems, where the system is relatively tractable yet clear challenges (methodological issues relating to soft core potentials, questions about why one series is predicted well while the other has a slope which is systematically off~\cite{calabro_elucidation_2016}, and dihedral sampling problems~\cite{calabro_accelerating_2015}) are still present. 
Another study performed relative free energy calculations between two ligands binding to thrombin and looked at dependence on starting ligand conformation, as well as force-field, finding substantial dependence on both. 
Enhanced sampling techniques reduced the dependance on starting ligand orientation~\cite{wang_achieving_2012}.


\subsection{Bromodomain proteins}

Bromodomains (BRDs) help read epigenetic markers and are present in many different human proteins; because of their role in transcription, they are being explored as potential drug targets in a range of clinical trials for various cancers and atherosclerosis~\cite{aldeghi_accurate_2016}. 
While little work with free energy calculations has focused on these proteins so far, the group of Philip Biggin from Oxford has recently begun studying these systems, as recently discussed at the 2016 Workshop on Free Energy Methods in Drug Design; their first study applying alchemical techniques to compute absolute binding free energies of 11 different ligands to BRD4 was recently published~\cite{aldeghi_accurate_2016}. 
This study is particularly interesting because it achieves a remarkable level of accuracy (an RMS error of $0.8\pm0.2$ kcal/mol) for absolute binding free energies spanning a range of 4 kcal/mol (with a magnitude up to $-9.8$ kcal/mol), and the relevant experimental results are high quality based on isothermal titration calorimetry. 
Even if crystal structures are not used as input and docking is substituted instead, the RMS error only degrades modestly. The compounds are relatively disparate, and not easily amenable to traditional relative free energy calculations. 
Thus, overall, this appears to be a case where free energy calculations can work quite well, in contrast to docking methods which were also examined. 
However, application of the techniques is not without challenges -- some ligands have multiple plausible binding modes, necessitating several calculations. 
This system would also pose severe challenges for relative free energy techniques.And work discussed at the Workshop noted above highlighted other challenges which are still to be resolved. 

\subsection{Cyclodextrins}
In the category of host-guest systems, cyclodextrins, and especially $\beta$ cyclodextrin, seem particularly interesting, possibly even as a near-term benchmark system. It is easily available, has good aqueous solubility, and has a wide variety of guests and host-derivatives which are easily accessible.  
Substantial work has already been done on $\beta$-cyclodextrin in particular~\cite{henriksen_computational_2015}; it has similar features to the other host-guest systems noted above, though in some cases some additional complexities such as two possible binding orientations that do not exchange on microsecond timescales~\cite{henriksen_computational_2015}, necessitating consideration of multiple possible binding modes. 
Parameters can also pose clear challenges there -- specifically, sodium and phosphate buffer has been observed to aggregate when using standard parameters at experimentally relevant conditions, so parameter modifications appear to be needed to adequately model experimental conditions~\cite{henriksen_computational_2015}. $\beta$ cyclodextrin has also been used to parameterize implicit solvent models explicitly for predicting binding interactions~\cite{wickstrom_parameterization_2016}.
\adddetail{This would need a careful look from Gilson as there is a lot to be said about cyclodextrins and I (DLM) don't know this literature like he does; he would need to expand.}

\subsection{Other systems of possible future interest}

Several other systems may be of possible interest because of the wealth and quality of experimental data, the extent of prior computational work, or the combination of pharmaceutical relevance with important challenges. 
None of these seem to be well-characterized or well-studied enough yet to be benchmark systems, but they may be interesting choices for the future. 
The first system we would put in this category is trypsin, and especially binding of benzamidine and its derivatives. 
While trypsin has been the focus of several free energy studies~\cite{talhout_understanding_2003, villa_sampling_2003, jiao_calculation_2008, jiao_trypsin_2009, de_ruiter_efficient_2012} and the SAMPL3 challenge~\cite{Newman:2011:JComputAidedMolDes, skillman_sampl3_2012}, it also appears to be subject to extremely slow protein motions (on the tens of microsecond timescale) which are only beginning to be characterized~\cite{plattner_protein_2015}, suggesting that short free energy calculations may appear to converge~\cite{talhout_understanding_2003, jiao_calculation_2008, jiao_trypsin_2009} but longer calculations can reveal a profound lack of convergence~\cite{plattner_protein_2015}. 
Coupled with work on Markov state modeling, trypsin may become a good benchmark for studying binding coupled with slow conformational dynamics~\cite{plattner_protein_2015}, but this will require more study to confirm.

HIV integrase may also be of potential interest as a benchmark system, in part because blind predictions for a set of fragments binding to several sites formed part of the basis of the SAMPL4 challenge~\cite{mobley_blind_2014, peat_interrogating_2014}. 
While the range of affinities in this specific set is relatively small, many crystal structures are available along with a wealth of verified nonbinders~\cite{peat_interrogating_2014}. 
One group actually did remarkably well using free energy calculations to recognize binding modes and predict nonbinders~\cite{gallicchio_virtual_2014, mobley_blind_2014}. 
Coupled with other HIV integrase data which is already available or becoming available, this may make this system attractive for future benchmarks.

FKBP has also been the focus of several different free energy studies on the same series of ligands over the years~\cite{shirts_calculating_2004, fujitani_direct_2005, jayachandran_parallelized-over-parts_2006, lee_calculation_2006, wang_absolute_2006, fujitani_massively_2009, ytreberg_absolute_2009}. 
However, in many cases, differences in calculated values between different studies -- even with what appears to be the same force field and system preparation -- are larger than differences relative to experiment~\cite{fujitani_direct_2005, jayachandran_parallelized-over-parts_2006}. 
Some challenges are clear, such as conformational sampling for some of the larger ligands~\cite{shirts_calculating_2004}, and there have been some suggestions of force field issues and long equilibration times for the protein~\cite{fujitani_massively_2009}. 
However, much more study is needed to be confident of the relevant issues before this will be suitable as a general-purpose benchmark system.

Some other systems which have been proposed include influenza neuraminidase inhibitors, some of which~\cite{smith_dihydropyrancarboxamides_1998} have been studied previously with some success~\cite{michel_protein_2006}, but other series include more complicated structure-activity relationships and protein loop motions~\cite{kerry_structural_2013}. 
Periplasmic oligopeptide binding protein A (OppA) may also be interesting as it binds a series of peptides two to five residues in length, regardless of composition; a large amount of calorimetric and crystallographic binding data is available~\cite{tame_crystal_1995, davies_relating_1999, sleigh_crystallographic_1999}, and the system appears challenging but potentially tractable for free energy calculations~\cite{maurer_calculation_2016}. 
JNK kinase may also pose interesting challenges for relative free energy calculations due to slow interconversion between binding modes for some ligands~\cite{kaus_how_2015}.


\section{THE FUTURE OF THIS WORK}
\label{sec:updates}
This work has so far presented a small set of benchmark systems for binding free energy calculations, highlighted some of the ways in which they have already proven their utility, and suggested some potential future benchmark systems. 
We hope that the community will become involved in identifying, characterizing, and helping to select additional benchmark systems, both from those proposed here as well as from systems which are currently being studied or which we have overlooked.
We seek community input to help characterize and identify suitable benchmark systems. 
We also expect that there may be community interest in test systems specifically selected to challenge sampling algorithms, without reference to experimental data.

In order to provide for updates of this material as new benchmark systems are defined, and to enable community input into the process of choosing them, we will make the LaTeX source for this article on GitHub at \url{http://www.github.com/mobleylab/benchmarksets}, with each version having a permanent DOI assigned by Zenodo. 
We encourage use of the GitHub issue tracker for discussion, comments, and proposed updates. 
We plan to incorporate new material via GitHub as one would for a coding project, then make it available via a preprint server, likely bioRxiv.
Given substantial changes to this initial version of the paper, it may ultimately be appropriate to make it available as a ``perpetual review''~\cite{mobley_proposal_2015} via another forum allowing versioned updates of publications. 

\section{CONCLUSIONS AND OUTLOOK}

Binding free energy calculations are a promising tool for predicting and understanding molecular interactions and appear to have enough accuracy to provide substantial benefits in a pharmaceutical drug discovery context.
However, progress is needed to improve these tools so that they can achieve their potential. To achieve steady progress, and to avoid potentially damaging cycles of enthusiasm and disillusionment, we need to understand and be open and honest about key challenges. 
Benchmarks are vital for this, as they allow researchers in the field to rigorously test their methods, arrive at a shared understanding of problems, and measure progress on well-characterized yet challenging systems. 
It is also worth emphasizing the importance of sharing information about apparently well thought-out and even promising methods that do \emph{not} work, rather than sharing only what does appear to work. 
Identifying and addressing failure cases and problems is critically important to advancing this technology, but failures can be harder to publish, and may even go unpublished, even though they serve a unique role in advancing the field. 
We therefore strongly encourage that such results be shared and welcomed by the research community.

Here, we proposed several benchmark systems for binding free energy calculations. 
These embody a subset of the key challenges facing the field, and we plan to expand the set as consensus emerges. 
Hopefully, these systems will serve as challenging standard test cases for new methods, force fields, protocols, and workflows. 
Our desire is that these benchmarks will advance the science and technology of modeling and predicting molecular interactions, and that other researchers in the field will contribute to identifying new benchmark sets and updating the information provided about these informative systems.


\section*{DISCLOSURE STATEMENT}
D.L.M. is a member of the Scientific Advisory Board for Schr\"{o}dinger, LLC. M.K.G. is a cofounder and has equity interest in the company VeraChem LLC.

\section*{ACKNOWLEDGMENTS}
DLM appreciates financial support from the National Institutes of Health (NIH; 1R01GM108889-01) and the National Science Foundation  (NSF; CHE 1352608). MKG thanks the NIH for partial support of this work through grant R01GM061300. The contents of this publication are solely the responsibility of the authors and do not necessarily represent the official views of the NIH or the NSF. 

We also appreciate helpful discussions with a huge number of people in the field, including a wide variety of participants at recent meetings such as the 2016 Workshop on Free Energy Methods in Drug Discovery.
Conversations with John Chodera (MSKCC), Chris Oostenbrink (BOKU), Julien Michel (Edinburgh), Robert Abel (Schr\"{o}dinger), Bruce Gibb (Tulane), Matt Sullivan (Tulane), and Lyle Isaacs (Maryland) were particularly helpful.

%Remaining todos
\listoftodos


\bibliographystyle{abbrv}
\bibliography{benchmarkset}




\end{document}
